### DISCUSSION TITLE
Unfollow users i havent followed

### DISCUSSION THREAD

**Discussion by dotslash** [community/community#15019] [Other Feature Feedback, Questions, & Ideas]
I want to unfollow activity of ex-coworkers' work at their new job

Scenario: 
2 months ago userX reviews a few PRs of mine on the repo of my current employer
userX switches job
Now I see activity of userX in their new job

**Comment by bennyfactor** [comment]
To add to this request, I believe this is not only a personal preference issue but also a trust & safety issue. My activity feed is highlighting the commits of an ex-coworker that left the organization on less-than-friendly terms. It is not hard to imagine a situation in which the highlights of an ex-coworker who was terminated for verbal abuse or other sorts of HR violations show in the feed of the person who was the target of that abuse.

**Comment by ghost** [comment]
+1, I would like to unfollow bot users. (Alternatively, please provide the ability for org-wide ATs so we don't have to create bot users.)

**Comment by JimmyMHames** [comment]
I'm with you on this one . Bot users are so dependant on the system to think for them.

**Comment by FilipKufalov** [comment]
Are you saying that he has access to the project you are working on even though he is no longer part of the same company?
